creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -20,11 +20,8 @@ const SyncMonitoringRetryDelayMs = 15000
 // Max number of attempts to select new replica set in reconfig
 const MAX_SELECT_NEW_REPLICA_SET_ATTEMPTS = 100
 
-// The number of nodes in a replica set
-const NUMBER_OF_REPLICA_SET_NODES = 3
-
 // Timeout for fetching batch clock values
-const BATCH_CLOCK_STATUS_REQUEST_TIMEOUT = 3000
+const BATCH_CLOCK_STATUS_REQUEST_TIMEOUT = 20000 // 20s
```

Comment:
don't need to change now but for future, i think its helpful to put the unit of measurement (ms in this case) in var name. more readable + less error prone

Reply:
sure, will note in future